### DISCUSSION TITLE
Setup a go2rtc stream clarification

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#5330] [General]
In the documentation "Configuring go2rtc" [https://deploy-preview-4055--frigate-docs.netlify.app/guides/configuring_go2rtc](url)

**Setup a go2rtc stream**
If you set the stream name under go2rtc to **match the name of your camera**, it will automatically be mapped.....

Assuming the path is already specified under 
cameras:
    Camera1:
      ffmpeg:
        inputs:
         path:   - rtsp://user:password@10.0.10.10:554/cam/realmonitor?channel=1&subtype=2

Does this mean we don't need to input the whole stream string but only the name of the camera?
from THIS:
go2rtc:
  streams:
    Camera1:
      - rtsp://user:password@10.0.10.10:554/cam/realmonitor?channel=1&subtype=2
to THIS:
go2rtc:
  streams:
    Camera1:
      - rtsp://Camera1

**Comment by NickM-27** [comment]
> Does this mean we don't need to input the whole stream string but only the name of the camera?
> from THIS:
> go2rtc:
> streams:
> Camera1:
> - rtsp://user:password@10.0.10.10:554/cam/realmonitor?channel=1&subtype=2
> to THIS:
> go2rtc:
> streams:
> Camera1:
> - rtsp://Camera1

No, it does not mean that. It is just saying if the names match then the live view will automatically use that camera in frigate frontend.
